internal/app/machined/pkg/controllers/network/route_spec_test.go

```diff
@@ -552,6 +552,123 @@ func (suite *RouteSpecSuite) TestLinkLocalRoute() {
 	suite.Require().NoError(suite.state.Destroy(suite.ctx, ll.Metadata()))
 }
 
+func (suite *RouteSpecSuite) TestLinkLocalRouteAlias() {
+	dummyInterface := suite.uniqueDummyInterface()
+	dummyAlias := suite.uniqueDummyInterface()
+
+	conn, err := rtnetlink.Dial(nil)
+	suite.Require().NoError(err)
+
+	defer conn.Close() //nolint:errcheck
+
+	suite.Require().NoError(
+		conn.Link.New(
```

Comment:
Won't these affect host? Is this a mock netlink or one actually managing the kernel? If latter is the case this test should be skipped when nl is not accessible (perhaps not root)

Reply:
It's already the case for all tests in this file, they all do the same thing